### PR TITLE
adds missing links for bylaws and mission/vision statements

### PR DESCRIPTION
**Pull request by jorydotcom**
Issues #230 and #322 note that the links in our CPC charter to the foundations Bylaws and mission/vision statements do not work. 

We point new applicants to these documents for drafting project submissions. This PR points applicants to what I believe are the best available documents to support their drafts.

### CODE REVIEW

File: CPC-CHARTER.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ and direction of the policies as described in the CPC Policy document and
 approved by the Board.
 
 The Board sets the overall CPC policy through [Foundation mission and vision
-statements][] as well as within the [Foundation bylaws][]. The policy describes
+statements][https://openjsf.org/about/] as well as within the [Foundation bylaws][https://bylaws.openjsf.org]. The policy describes
```
Comment (ljharb) [inline CPC-CHARTER.md]: Rather than adding these inline here; if you update the x’s on lines 312-313, they’ll also work

Comment (jorydotcom) [reply CPC-CHARTER.md]: This has been updated per your requested changes.

**Comment by mhdawson** [comment]
Sent request to start vote by the Board.

**Comment by mhdawson** [comment]
This was approved by the board on Friday.

**Comment by mhdawson** [comment]
As a charter change needs to have been open for 14 days (see https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#aproving-project-charters)

Will plan to land tomorrow if there are no objections before then.

**Comment by mhdawson** [comment]
Passed the 14 days required by our governance.  Landing

**Pull request merged**